package.json

```diff
@@ -12,14 +12,15 @@
     "dev:library": "babel src -w -d lib",
     "build:styleguide": "build-storybook",
     "build:library": "babel src -d lib",
-    "verify": "yarn lint && yarn build:library && yarn build:styleguide",
+    "preverify": "yarn lint && yarn build:library",
```

Comment:
the storybook as a precommit hook felt brutal and not really needed, so I did this to make it less brutal